app/Http/Controllers/Admin/Settings/MailController.php

```diff
@@ -111,4 +115,26 @@ public function update(MailSettingsFormRequest $request): RedirectResponse
 
         return redirect()->route('admin.settings.mail');
     }
+
+    /**
+     * Submit a request to send a test mail message.
+     *
+     * @param Request $request
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function test(Request $request): RedirectResponse
+    {
+        try {
+            Notification::route('mail', $request->user()->email)
+                ->notify(new MailTested($request->user()));
+        } catch (Exception $exception) {
+            $this->alert->danger(trans('base.mail.test_failed'))->flash();
```

Comment:
Think it makes sense to catch some type of mailer exception and display the message here? Might help us save a step when the user comes asking saying 'There was an error'.